requirements.txt

```diff
@@ -4,4 +4,5 @@
 
 mcp>=1.0.0
 httpx>=0.25.0
+anyio>=3.0.0
```

Comment:
Prior review indicated this dependency was not needed.  Can it be removed from this file and from the toml file?